neurolib/models/multimodel/builder/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .aln import ALNNetwork, ALNNode
+from .base.network import Network, Node, SingleCouplingExcitatoryInhibitoryNode
+from .base.neural_mass import NeuralMass
+from .fitzhugh_nagumo import FitzHughNagumoNetwork, FitzHughNagumoNode
+from .hopf import HopfNetwork, HopfNode
+from .thalamus import ThalamicNode
+from .wilson_cowan import WilsonCowanNetwork, WilsonCowanNode
+from .wong_wang import ReducedWongWangNetwork, ReducedWongWangNode, WongWangNetwork, WongWangNode
```

Comment:
just to import building blocks directly from `neurolib.models.multimodel.builder`